src/plugins/discover/public/application/components/default_discover_table/table_row.tsx

```diff
@@ -48,7 +48,7 @@ const TableRowUI = ({
   ]);
 
   const tableRow = (
-    <tr key={row._id}>
+    <tr key={row._id} className={row.isAnchor ? 'osdDocTable__row--highlight' : ''}>
```

Comment:
Is there a test for this or is it covered in some other way?

Reply:
I don't think default discover table is covered by any unit tests. src/plugins/discover/public/application/components/default_discover_table

Reply:
Should we add a ftr test? cuz we want to gradually build up the test coverage.

Reply:
I think we want to add unit tests and functional tests for default discover table overall. Created an issue to track it here: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/7048